sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/SessionTokenHelper.java

```diff
@@ -57,8 +57,9 @@ public static void setPartitionLocalSessionToken(RxDocumentServiceRequest reques
         if (request.requestContext.sessionToken == null) {
             request.getHeaders().remove(HttpConstants.HttpHeaders.SESSION_TOKEN);
         } else {
+
             request.getHeaders().put(HttpConstants.HttpHeaders.SESSION_TOKEN,
-                                     String.format("%1s:%2s", partitionKeyRangeId, request.requestContext.sessionToken.convertToString()));
+                                     concatPartitionKeyRangeIdWithSessionToken(partitionKeyRangeId, request.requestContext.sessionToken.convertToString()));
```

Comment:
Glad we made this change. We should make sure to populate this change everywhere, specially in `OrderByDocumentQueryExecutionContext`.

Reply:
The scope of current PR is only single point read. I will investigate the query separately.